tutorials/katacoda/thanos/2-lts/step3.md

```diff
@@ -0,0 +1,73 @@
+# Step 3 - Installing the Thanos Store
+
+In this step, we will learn about Thanos Store Gateway, how to start, and what problems are solved by it.
+
+## Thanos Components
+
+Thanos is a single Go binary capable to run in different modes. Each mode represents a different component and can be invoked in a single command.
```

Comment:
I think we repeat oursevles here a bit. Wonder if we can go straight to --help and store